src/features/CourtCaseFilters/FilterChipSection.tsx

```diff
@@ -123,6 +128,22 @@ const FilterChipSection: React.FC<Props> = ({ state, dispatch, sectionState, mar
         value={state.dateFilter.value!}
       />
 
+      <FilterChipRow
+        chipLabel={customDateRangeLabel}
+        condition={
+          state.customDateFrom.value !== undefined &&
+          state.customDateTo.value !== undefined &&
+          state.customDateFrom.state === sectionState &&
+          state.customDateTo.state === sectionState
+        }
+        dispatch={dispatch}
+        type="customDate"
+        label="Custom date range"
+        // TODO: check behaviour if only 1 field set.
```

Comment:
Currently, a custom date range only sets the filter if both field has a value, I think this is acceptable for an MVP, so we can remove the comment

Reply:
Removed